components/SSI/SBTxWallet/UpdateUsername/index.tsx

```diff
@@ -42,10 +47,48 @@ function Component() {
         key,
     }: React.KeyboardEvent<HTMLInputElement>) => {
         if (key === 'Enter') {
-            setSaved(true)
+            handleSave()
         }
     }
 
+    const handleSave = async () => {
+        setLoading(true)
+        const addr = await tyron.SearchBarUtil.default.fetchAddr(
```

Comment:
Hi @devwistkey, to verify that a username is valid, it is enough to fetchAddr(username, 'did')

Reply:
noted